lib/src/current_weather/presentation/pages/current_weather_page.dart

```diff
@@ -22,6 +24,19 @@ class CurrentWeatherPage extends HookConsumerWidget {
     );
     final currentWeather = ref.watch(currentWeatherProvider);
 
+    ref.listen(connectivityStreamProvider, (_, next) {
+      final data = next.asData;
+      if (data == null) return;
+
+      final hasInternet = data.value != ConnectivityResult.none;
+
+      final message = hasInternet ? 'You are online' : 'You are offline';
+      final color = hasInternet ? Colors.green : Colors.red;
+
+      final snackBar = SnackBar(content: Text(message), backgroundColor: color);
```

Comment:
`SnackBar`s tend to have a long delay before they disappear. You can change that easily if you tamper with its properties a bit. Could you do that?

I'm actually not sure if that can be changed into `SnackBar` or into `showSnackBar` directly.